addons/project/static/src/views/project_collaborator_list/project_collaborator_list_view.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+import { listView } from '@web/views/list/list_view';
+import { InviteCollaboratorsListController } from './project_collaborator_list_controller';
+
+export const ProjectCollaboratorsListView = {
+    ...listView,
+    Controller: InviteCollaboratorsListController,
```

Comment:
```suggestion
    Controller: ProjectCollaboratorsListController,
```
same here :confused: